### DISCUSSION TITLE
Hot to import local build of DaisyUI

### DISCUSSION THREAD

**Discussion by flomero** [saadeghi/daisyui#2370] [Q&A] [answered]
After editing the source files and running `npm run build` how do i import the local dist folder correctly into a project to test it?
Somehow can't get it to work and always get postcss errors like `Internal server error: [postcss] Cannot find module`

**Comment by saadeghi** [comment]
After building the package, if you `require` the local path of your build, it should work.  
Also don't forget to `npm i -D daisyui@latest` on your project so you will have the dependencies in your project.

If it's not working, please share more info about your setup so I can reproduce the issue.

**Reply by flomero** [reply]
do i have to link to a specifiy file or just `require('../daisyui/dist')`?

**Reply by saadeghi** [reply, marked as answer]
@flomero `require('/path/to/daisyui/src')` ([where `index.js` is](https://unpkg.com/browse/daisyui@3.8.0/src/))  

(I don't know if the relative path should be`../` or not. it depends where is your daisyUI build and where is you'r project's `tailwind.config.js` file)

**Reply by flomero** [reply]
that's it thanks 🙌🏼. tried to somehow require the `dist`  folder instead of `src`. created a pull request updating the contributing guide accordingly https://github.com/saadeghi/daisyui/pull/2373

**Reply by saadeghi** [reply]
@flomero Thank you for the PR
